src/js/components/explorer/detail/sidebar/ExplorerSidebar.jsx

```diff
@@ -107,11 +118,11 @@ export default class ExplorerSidebar extends React.Component {
                     </a>
                 </div>
 
-                <QuarterPicker
-                    fy={this.props.fy}
+                <QuarterPickerWithFY
+                    selectedFy={this.props.fy}
                     quarter={this.props.quarter}
```

Comment:
This is broken b/c this component is expecting `quarter` to be a true quarter value rather than a period value.